specification/cognitiveservices/data-plane/Speech/BatchAvatar/preview/2024-04-15-preview/batchavatar.json

```diff
@@ -0,0 +1,846 @@
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Detailed error code to help diagnostic.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The source of the error.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Detailed error message.",
+          "type": "string"
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError"
+        }
+      }
+    },
+    "OperationId": {
+      "description": "Unique operation ID.",
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9][a-zA-Z0-9._-]{1,62}[a-zA-Z0-9]$",
+      "minLength": 3,
+      "maxLength": 64
+    },
+    "Operation": {
+      "description": "Status of a long running operation.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/OperationId"
+        },
+        "status": {
+          "$ref": "#/definitions/OperationStatus",
+          "readOnly": true
+        }
+      }
+    },
+    "OperationStatus": {
+      "description": "Status of an operation.",
+      "enum": [
+        "NotStarted",
+        "Running",
+        "Succeeded",
+        "Failed"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "OperationStatus",
+        "modelAsString": true
+      }
+    },
+    "PaginatedBatchSynthesis": {
+      "description": "Paginated batch synthesis task list",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Bath synthesis tasks.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BatchSynthesis"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page.",
+          "format": "uri",
+          "type": "string"
+        }
+      }
+    },
+    "ResourceId": {
+      "description": "Resource id",
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9][a-zA-Z0-9._-]{1,62}[a-zA-Z0-9]$",
+      "minLength": 3,
+      "maxLength": 64
+    },
+    "Status": {
+      "description": "Status of a resource.",
+      "enum": [
+        "NotStarted",
+        "Running",
+        "Succeeded",
+        "Failed"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "Status",
+        "modelAsString": true
+      }
+    },
+    "InputKind": {
+      "description": "Input kind.",
+      "enum": [
+        "PlainText",
+        "SSML"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "InputKind",
+        "modelAsString": true
+      }
+    }
```

Comment:
nit for future, Speech may want to put the common definitions to a single place.

I remember a `InputKind` at https://github.com/Azure/azure-rest-api-specs/pull/28272/files#r1542276805

Reply:
Yes they are born from the same structure.
For now we are not sure how they may evolve in future yet: They may go differently to have different supported `inputKind`.
But yeah we'll consider merging these shared things in future version.